addons/web/static/src/core/notebook/notebook.scss

```diff
@@ -74,4 +74,13 @@
             }
         }
     }
+
```

Comment:
Mini remarks concerning the commit message (just grammar and stuff, the content is great):

- The "commandbar" is called "Powerbox" ^^'
- "there was functional issues" -> "there _were_ functional issues"
- "responsivenessof" -> "responsiveness of"
- "In a old commit (`<url>`)" -> "In commit [1]" (and then at the end of the commit message, add "[1]: `<url>`")
- "enought" -> "enough"
- "on the all the editor toolbars" -> "on all editor toolbars" (or "on all toolbars of the editor" (@dmo-odoo ! Found one in the wild!)
- "an horizontal scroll" -> "a horizontal scroll"